woocommerce/woocommerce-android
# Add simple screenshots

**To Test**

1. Run bundle install --with screenshots && bundle exec fastlane create_promo_screenshots, and if prompted, confirm that you'd like to overwrite existing promo screenshots.
2. Ensure that the generated screenshots look ok.
